### ISSUE TITLE
in-text citations not ordered chronologically in conservation-biology.csl

### ISSUE THREAD

**Issue by DanOvando** [citation-style-language/styles#5146]
The [style guide for Conservation Biology](https://conbio.onlinelibrary.wiley.com/pb-assets/assets/15231739/Author%20Style%20Guide%20feb2019-1551741575403.pdf) states that "In the body of the paper, order citations from oldest to newest and use author-year format. "

However, when I use the conservaiton-biology.csl file in this repo, the rendered in-text citations are not in chronological or alphabetical order, or the order in which they were entered. 

Changing 
```
    <sort>
      <key variable="issued"/>
      <key macro="author"/>
    </sort>
```

at
https://github.com/citation-style-language/styles/blob/a0da11e57b1747307b5291a18844f7d07c585fb5/conservation-biology.csl#L193

to 
```
    <sort>
      <key macro="issued"/>
      <key variable="author" sort="ascending"/>
    </sort>
```

Seems to solve the problem and produces in-text citations sorted first by year then by first author in alphabetical order . But, my .csl skills are close to non-existent, and this fix was just from a bunch of trial and error, but it works

**Comment by adam3smith** [comment]
Which reference manager? The current sort macro should do this correctly & does for me in Zotero. Using macro vs. variable issued should matter very little (the macro would only use the year for some item types, the variable always the full date)

**Comment by DanOvando** [comment]
Ah sorry should have specified. References exported by zotero to a .yaml file using Better BibTeX. Same problem occured when exporting to a .bib file. 

Then manually entering citations into a .Rmd via 

`Here are some citations [@hastings2003;@gerber2003;@hilborn2004a;@gaines2010;@botsford2003;@smith2003a;@white2011;@ovando2016;@moffitt2013;@walters2000a]`

Then knitting to PDF via bookdown::pdf_document2. Same problem persists when knitting to Word or HTML

When knit, that walters2000a citation which should be at the beginning would instead end up somewhere in the middle for reasons unknown to me. Changing the conservation-biology.csl file with the changes I included above results in @walters2000a now being rendered at the start of the list, and all other citations in chronological then alphabetical order. 

Apologies, I'm not sure how to submit a reprex for something like this, but I can try if needed

**Comment by DanOvando** [comment]
Using the current version of styles/conservation-biology.csl, knitting

`Here are some citations [@hastings2003;@gerber2003;@hilborn2004a;@gaines2010;@botsford2003;@smith2003a;@white2011;@ovando2016;@moffitt2013;@walters2000a]`

Produces "Here are some citations (Hilborn et al. 2004; White et al. 2011; Moffitt et al. 2013; Walters et al. 2000; Botsford et al. 2003; Gerber et al. 2003; Hastings & Botsford 2003; Smith & Wilen 2003; Gaines et al. 2010; Ovando et al. 2016)

Using 
```
    <sort>
      <key macro="issued"/>
      <key variable="author" sort="ascending"/>
    </sort>
```

Produces

"Here are some citations (Walters et al. 2000; Botsford et al. 2003; Gerber et al. 2003; Hastings & Botsford 2003; Smith & Wilen 2003; Hilborn et al. 2004; Gaines et al. 2010; White et al. 2011; Moffitt et al. 2013; Ovando et al. 2016)"

Certainly not claiming that my .csl change is inherently correct, but seems to get the job done, so curious as to what's going on here

**Comment by adam3smith** [comment]
That looks like a citeproc-pandoc bug to me (that's the CSL processor used during knitting from Rmd). Would be good to verify if this is present using an up to date version of pandoc and the report to https://github.com/jgm/pandoc-citeproc/ via a minimal working example. Feel free to point to this discussion for context.

I think it's pretty clear that the style is correct, so I don't think we'll adjust that. Thanks for narrowing this down!

**Comment by DanOvando** [comment]
Aha, upgrading to pandoc 2.11.2 fixed it (with the CSL as is), I was on 2.9 something before, I wonder if that's what ships with the preview version of RStudio since I don't recall manually installing pandoc. Sorry about that, thanks for taking the time to look at this (and for maintaining this, Zotero + R Markdown is such an amazing time saver).
